### DISCUSSION TITLE
post/zustand-refactoring/

### DISCUSSION THREAD

**Discussion by giscus** [jgjgill/blog#20] [Comments]
# post/zustand-refactoring/

프로젝트 내 Zustand 관련 로직을 리팩터링한 경험을 기록합니다.

https://jgjgill.com/post/zustand-refactoring/

**Comment by AnJeMin** [comment]
안녕하세요. 좋은 글 작성해주셔서 감사합니다.
Zustand의 메인테이너가 작성한 책으로 스터디를 진행하셨다고 하셨는데, 혹시 책 이름 공유받을 수 있을까요??

**Reply by jgjgill** [reply]
@AnJeMin 안녕하세요!
[리액트 훅을 활용한 마이크로 상태 관리](https://product.kyobobook.co.kr/detail/S000212233308) 책으로 스터디했습니다..! 🙇‍♂️
